ci/travis-36-numpydev.yaml

```diff
@@ -0,0 +1,18 @@
+name: pandas
+channels:
```

Comment:
it seems that you added conda-forge to virtually all recipes. We did not have that before. See if you can remove it from a few (IOW just defaults)

Reply:
Will do. I needed it for moto, but it shouldn’t be required / used here, since s3fs isn’t installed.